spring-cloud-dataflow-server-local/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  kafka:
+    image: wurstmeister/kafka:0.10.1.0
+    expose:
+      - "9092"
+    environment:
+      - KAFKA_ADVERTISED_PORT=9092
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+    depends_on:
+      - zookeeper
+  redis:
+    image: redis:2.8
+    expose:
+      - "6379"
+  zookeeper:
+    image: wurstmeister/zookeeper
+    expose:
+      - "2181"
+    environment:
+      - KAFKA_ADVERTISED_HOST_NAME=zookeeper
+
+  dataflow-server:
+    image: springcloud/spring-cloud-dataflow-server-local:latest
```

Comment:
Right now, given the current state of the image, the Dashboard fails to load with the following error. It'd be better to actually pull the latest GA release to avoid this type of issues.

![image](https://user-images.githubusercontent.com/1562654/36803276-f51c76e2-1c6b-11e8-8f17-c1de10c5caa8.png)

Reply:
yup thats fine. what is the typical version bumping process, for example i just used "latest" in this first pass to get something out on docker hub without committing to any versioning. 

lets say today we have (made up tag names):

`image: springcloud/spring-cloud-dataflow-local:1.3.0.RELEASE`

then tomorrow we release 1.3.1 - would the typical process be for somebody to come into this file and manually change `1.3.0.RELEASE` to `1.3.1.RELEASE` or do we have some kind of versioning templating system as I would think this may apply in other areas too?